python/pyhail/dataset.py

```diff
@@ -251,13 +297,38 @@ def annotate_samples_fam(self, input, quantpheno=False, delimiter='\\\\s+', root
         return self.hc.run_command(self, pargs)
 
     def annotate_samples_list(self, input, root):
-        """Annotate samples with a Boolean indicating presence/absence in a
-        list of samples in a text file.
+        """Annotate samples with a Boolean indicating presence/absence in a list of samples in a text file.
+
+        **Example**
+
+        We have a file `batch1.txt` with a list of samples::
```

Comment:
I don't think you need to include the example file here.  A bit long.  We should just make sure it is easy to access the example files (e.g. create links in the code if that is possible).